### ISSUE TITLE
Add contributation guideline file

### ISSUE THREAD

**Issue by fho** [rtrlib/rtrlib#2]
Add a file that contains rules for contributing to rtrlib.
* Coding Style (kernel code style)
* commit message
* How to send a patch (github pull request, git format-patch)

**Comment by waehlisch** [comment]
Have a look [here](https://github.com/rtrlib/rtrlib/wiki/Contributing-to-RTRlib).

**Comment by smlng** [comment]
Look good to me. Should we create a file `CONTRIBUTING` with same content or with link to the WIKI within the repo root?

**Comment by fho** [comment]
CONTRIBUTING file is a good idea :+1:

**Comment by smlng** [comment]
I looked into some other projects and their contributing policies/description. I found that some also state that new features or extended modification should first be discussed with the community via an issue.

Maybe we should add some text on that too. Stating that a PR should address/reference a certain issue if it introduces major code changes. If no such issue exists, the user must create one first to start a discussion on that topic _before_ implementing stuff. Maybe some issue is not an issue at all, or not seen as one by other rtrlib members; and some _bugs_ could be features :-)

However, this rule does not apply to PRs with only minor code changes (typos, simple bug fixes). These are allowed to be submitted without referring to an issue.
